src/nvim/api/vim.c

```diff
@@ -690,6 +690,97 @@ void nvim_set_current_win(Window window, Error *err)
   }
 }
 
+/// Create new buffer with requested buftype
+///
+/// 
+/// For instance, swap file prompt is only shown then.
```

Comment:
Copy paste error on this line?

Reply:
Yes, line was only relevant when function could open existing files.